src/cli.coffee

```diff
@@ -4,55 +4,90 @@
 {
   discover
   reinstall_all
+  shrinkwrap
 }         = require '.'
 cli       = require 'commander'
 path      = require 'path'
+fs        = require 'fs'
 
 cli
   .version '0.0.0' # TODO: Read from package.json
   .description """
     A utility to properly install npm git dependencies.
   """
-  .arguments '<file>'
   .option '-s --silent',  'suppress child processes output'
+  .option '-w, --git_shrinkwrap <path>', 'Optional git shrinkwrap file location'
+  .option '-c, --package <path>', 'Optional git shrinkwrap file location'
   .option '-v --verbose', 'be verbose'
-  .option '-a --all',     'reinstall all git dependencies and devDependencies'
+  #.option '-a --all',     'reinstall all git dependencies and devDependencies'
```

Comment:
This option was never implemented in the original code, so I pulled it and the stubbed code below, since I didn't want to have to refactor a thing that didn't have an implementation yet.